### PR TITLE
Add partial translation of functionality section

### PR DESCRIPTION
**Pull request by jlcastrogro**
Signed-off-by: José Luis Castro Guerrero <jlcastrogro@gmail.com>

**Description**

This PR fixes partially #1776

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by jesuslerma — CHANGES_REQUESTED** [review]
Hi @jlcastrogro you should add the `es` to permalink. Please ensure to have this `es/functionality` to `es/functionality.md` file. And for `docs/pages/es/functionality/management.md` add `es/functionality/lifecycle-managemen` to permalink. 

Please also ensure to not translate the word `service mesh`.

**Comment by jlcastrogro** [comment]
@jesuslerma I will do it. And what should to do next?

File: docs/pages/es/functionality/management.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+# Gestión del ciclo de vida de la malla de servicio
+Meshery soporta varias [mallas de servicios diferentes](installation/adapters), de modo que puede aprender y comparar diferentes funcionalidades de malla de servicio. Una vez que haya seleccionado operar una mallas de servicios determinado, Meshery le ayudará con las actividades en curso.
+
+## Despliegue de una malla de servicios
+Meshery automatiza el aprovisionamiento de varias mallas de servicios, permitiendo a los usuarios seleccionar diferentes perfiles de configuración para admitir una variedad de modelos de despliegue.
+
+## Instalación de aplicaciones de muestra
+Comúnmente junto a cada malla de servicios hay una aplicación de muestra que facilita la demosatración del valor de la malla de servicios dada. Meshery permite que rapidamente despliegue la misma aplicación de muestra en diferentes mallas de servicios.
```
Comment (edwvilla) [inline docs/pages/es/functionality/management.md:15]: typo: demostración

**Review by edwvilla — CHANGES_REQUESTED** [review]

File: docs/pages/es/functionality/management.md (hunk 1 above)
Comment (edwvilla) [reply docs/pages/es/functionality/management.md:15]: everything else looks good, great work mate 👍

Comment (jlcastrogro) [reply docs/pages/es/functionality/management.md:15]: Hi, @edwvilla, Do I need to change the type too?

**Comment by jlcastrogro** [comment]
@jesuslerma I already made the changes, what should I do now?

**Comment by rubenlorenzo** [comment]
> @jesuslerma I already made the changes, what should I do now?

 Hi, @jlcastrogro would have that to do another commit with the changes. One question @jesuslerma, would be necessary to do changes in another branch? I also see very well, the translation.

**Comment by jlcastrogro** [comment]
@rubenlorenzo So I close this PR and open a new one with the changes?, thanks

**Comment by rubenlorenzo** [comment]
About commits to a new branch is just a question. Maybe it is not necessary. It is just to know the opinion of @jesuslerma.

File: docs/pages/es/functionality/management.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Lifecycle Management
+permalink: es/functionality/lifecycle-management
+type: functionality
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/management.md:5]: In type would put: _demostracion_ and also in the next documents.

File: docs/pages/es/functionality/performance.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Performance Management
+permalink: es/functionality/performance-management
+type: functionality
+---
+
+# Gestión del rendimiento
+La clave del funcionamiento eficiente de cualuier service mesh es la medición y gestión de su rendimiento.
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/performance.md:9]: In the next sentences, some letter is missing. These words are in italics: _cualquiera_

File: docs/pages/es/functionality/performance.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+NightHawk es una herramienta de caracterización L7 (HTTP/HTTPS/HTTP2). Actualmente ofrece:
+
+- Un cliente de prueba de carga que soporta HTTP/1.1 y HTTP/2 sobre HTTP y HTTPS (Los certificados HTTPS aún no se validan).
+- Un servidor de prueba simple capaz de generar tamaños de respuestas dinámicas, así como también inyectar retrasos.
+- Un binario para tranformar la salida de NightHawk a formatos conocidos, permitiendo la integración con otros sistemas y paneles.
+
+## Node y métricas del service mesh
+
+Meshery proporciona los resultados de pruebas de rendimiento junto a las méticas del entorno, incluyendo el control del service mesh y las métricas del plano de datos. También incluye las métricas de los recursos del nodo del clúster, para que los operadores puedan comprender fácilmente la sobrecarga del plano de control y el plano de datos de su service mesh en el contexto de la sobrecarga incurrida en los nodos dentro del clúster.
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/performance.md:39]: junto a las _métricas_

File: docs/pages/es/functionality/performance.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+### Conexión a Grafana
+Si tiene una clave de API configurada para restringir el acceso a sus tableros de Grafana, deberá ingresar la clave de API cuando establezca la conexión de Meshery con Grafana.
+
+* Importación de tableros de Grafana
+    - Importación de tablero existente de Grafana a través de API
+    - Importación de tablero personalizado de Grafana a través de yaml
+* Configuración de las preferencias del panel de gráficos
+
+## Prometheus y Meshery
+Meshery permite a los usuarios conectarse a una o más instancias de Prometheus para recopilar datos telemétricos (en forma de métricas). Estas métricas pueden pertenecer al service mesh, Kubernetes, aplicaciones en la malla o realmente... cualquier mátrica que Prometheus haya recolectado.
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/performance.md:58]: ... cualquier _métrica_

File: docs/pages/es/functionality/smi-conformance.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+## Validación de conformidad
+
+La conformidad con las especificaciones de SMI se harán a través el aprovisionamiento de service mesh individuales y el despliegue de una carga de trabajo en común. se utiliza una aplicación de muestra sencilla e instrumentada como carga de trabajo para probar.
+
+## Definición de conformidad
+
+La conformidad con las especificaciones de SMI es definida como una serie de pruebas de aserción. Una prueba de aserción es una condición que debe probarse para confirmmr la conformidad con un requisito. Una prueba de aserción es una condición que desde la perspectiva de las pruebas de validación, determinan la conformidad requerida para que se prueben varias condiciones. La colección de pruebas de aserción clasificadas por la especificación de SMI define colectivamente el conjunto de pruebas de conformidad de SMI. Meshery es el mejor arnés de prueba utilizado para ajustar las pruebas de conformidad de SMI para diferentes service mesh y diferentes cargas de trabajo.
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/smi-conformance.md:57]: debe probarse para _confirmar_

File: docs/pages/es/functionality/smi-conformance.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+## Pasos para ejecutar pruebas de rendimiento
+
+### Pre condiciones
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/smi-conformance.md:61]: title: _Precondiciones_

File: docs/pages/es/functionality/management.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/es/functionality/management.md]: @edwvilla thank you very much for these reviews and your contributions. 👌

**Comment by leecalcote** [comment]
@rubenlorenzo its so nice to have you engaged here. 🙂 Thank you.

**Comment by jlcastrogro** [comment]
@rubenlorenzo thanks for the comments.

**Comment by rubenlorenzo** [comment]
You're welcome @leecalcote and @jlcastrogro :+1: , I hope to be more attentive. I've been a little busy.

File: docs/pages/es/functionality/smi-conformance.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+## Conformidad
+
+Definiendo “Conformidad” - Es importante reconocer que la conformidad consiste en capacidades y estado de conformidad. Nosotros definimos la conformidad como una combinación de estos 2 conceptos.
+
+1. La conformidad de SMI reconoce que
+   ...algunos service mesh participantes nunca implementan completamente funciones a conciencia (especificaciones SMI)...
+
+2. La conformidad SMI identifica
+   ...una diferencia entre la completa implementación de una espacificación y el cumplimiento de las partes que implementa...
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/smi-conformance.md:18]: I have found the following: de una _especificación_

File: docs/pages/es/functionality/smi-conformance.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+Cada versión de Kubernetes, versión de service mesh y categoría de SMI serán sometidos a varias pruebas. Cada prueba se llevará a cabo de forma automatizada y concurrente, principalmente invocando a Meshery para ejecutar las pruebas de conformidad.
+
+#### Estos son los pasos
+
+- Configurar una versión específica de un service mesh
+- [Opcional] Configurar Prometheus
+- Desplegar una aplicación de muestra elegida
+- Desplegar el operador SMI necesario, como smi-metrics
+  la prueba se ejecutar+a, lo que puede implicar la llamada a un punto final de API
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/smi-conformance.md:40]: _ejecutará_

File: docs/pages/es/functionality/smi-conformance.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+La conformidad con las especificaciones de SMI es definida como una serie de pruebas de aserción. Una prueba de aserción es una condición que debe probarse para confirmar la conformidad con un requisito. Una prueba de aserción es una condición que desde la perspectiva de las pruebas de validación, determinan la conformidad requerida para que se prueben varias condiciones. La colección de pruebas de aserción clasificadas por la especificación de SMI define colectivamente el conjunto de pruebas de conformidad de SMI. Meshery es el mejor arnés de prueba utilizado para ajustar las pruebas de conformidad de SMI para diferentes service mesh y diferentes cargas de trabajo.
+
+## Pasos para ejecutar pruebas de rendimiento
+
+### Precondiciones
+
+- La capacidad de un determinado service mesh para adherirse a una especificación de SMI es validada ejecutando una carga de trabajo sobre el service mesh.
+- Los depliegues de cargas de trabajo son configurados de forma específica para los requisitos de incorporación del service mesh dado.
+- Las pruebas son definidas para validar la conformidad para cada tipo de especificación de SMI (por ejemplo, métricas, acceso, tráfico,...).
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/smi-conformance.md:65]: _tráfico..._

File: docs/pages/es/functionality/smi-conformance.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+- Las pruebas de aserción son definidas de acuerdo a una carga de trabajo específica y son desplegadas con las cargas de trabajo que se están probando (las pruebas de aserción están empaquetadas).
+- Un resultado de prueba es recolectado con la evaluación de cada aserción.
+- Los resultados de futuras pruebas se transmitirán individualmente a Meshery después de que se evalúe cada aserción.
+- Una vez que todas las aserciones son evaluadas, los resultados de las pruebas son retornadas a Meshery para su representación visual.
+
+## Informe de conformidad
+
+### Procedencia de los resultados de pruebas
+
+Cada proyecto de service mesh participante se le pedirá que incorpore la herramienta de conformidad, Meshery, en sus pipelines de CI o, alternativamente, que ejecute manualmente el conjuntto de pruebas de conformidad cuando se realice un despliegue del proyecto de service mesh. La herramienta de conformidad ejecutará el conjunto de pruebas y automáticamente actualizará el panel de conformidad cuando la prueba sea completada.
```
Comment (rubenlorenzo) [inline docs/pages/es/functionality/smi-conformance.md:78]: manualmente el _conjunto_

**Comment by jesuslerma** [comment]
Hi @rubenlorenzo at this moment is not required to have a specific branch name. @jlcastrogro  please just fix the last comments that @rubenlorenzo  left.

File: docs/pages/es/functionality/management.md (hunk 2 above)
Comment (jesuslerma) [reply docs/pages/es/functionality/management.md]: hi @rubenlorenzo  I think functionality (the valie for type) is an internal jekyll stuff, so it should not be changed. In the path `es/functionality/lifecycle-management` Iḿ not sure if it should be changed. Weŕe going to dig into #1789 to see how that plugin works, but at this moment these changes are not required.

Comment (rubenlorenzo) [reply docs/pages/es/functionality/management.md]: Agree @jesuslerma, I had in mind the comment @edwvilla. I will follow issue #1789.

**Comment by rubenlorenzo** [comment]
@jesuslerma. I feel well, having helped.

File: docs/pages/es/functionality/management.md (hunk 2 above)
Comment (jlcastrogro) [reply docs/pages/es/functionality/management.md]: @jesuslerma I had already applied this change, Do I put in **type** _functionality_ again?

Comment (edwvilla) [reply docs/pages/es/functionality/management.md]: Oh, i meant he had a typo on `demostración` didn't mean to change the jekyll type, sorry for the confusion 😅

**Comment by leecalcote** [comment]
Wow! Nice.

**Review by jesuslerma — APPROVED** [review]
Awesome work. All is looking good to me. Thanks @jlcastrogro !

**Review by leecalcote — APPROVED** [review]

**Pull request merged**